tests/draft-next/unevaluatedProperties.json

```diff
@@ -1603,5 +1603,65 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description": "propertyDependencies with unevaluatedProperties" ,
+        "schema" : {
+            "$schema": "https://json-schema.org/draft/next/schema",
+            "propertyDependencies": {
+                "foo": {
+                    "bar": {
+                        "properties": {
+                            "buz": {}
+                        }
+                    }
+                }
+            },
+            "unevaluatedProperties": false
+        },
+
+        "tests": [
+            {
+                "description": "unevaluatedProperties doesn't consider propertyDependencies" , 
+                "data": {"foo": "bar"},
+                "valid": false
+            },
+            {
+                "description": "unevaluatedProperties doesn't consider inner schema properties",
+                "data": {"foo": "bar", "buz": ""},
+                "valid": false
+            }
+        ]
+    },
+    {
+        "description": "dependentSchemas with unevaluatedProperties",
+        "schema": {
+            "$schema": "https://json-schema.org/draft/next/schema",
+            "dependentSchemas": {
+                "foo": {
+                    "properties": {
+                        "bar":{}
+                    }
+                }
+            },
+            "unevaluatedProperties": false
+        },
+        "tests": [
+            {
+                "description": "unevaluatedProperties doesn't consider dependentSchemas",
+                "data": {"foo": ""},
+                "valid": false
+            },
+            {
+                "description": "unevaluatedproperties does't consider inner schema property",
+                "data": {"bar": ""},
+                "valid": false
+            },
```

Comment:
👍  This is a good test because the `/propertyDependencies/foo` subschema is never evaluated, so `bar` wouldn't be allowed.  We should also add this test case to the `propertyDependencies` scenario above.